src/main/java/service/CsvFileReader.java

```diff
@@ -0,0 +1,19 @@
+package service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class CsvFileReader implements FileReader {
```

Comment:
we can actually read any file with this implementation
```suggestion
public class FileReaderImpl implements FileReader {
```